src/app/containers/StoryPromo/MediaIndicator/index.jsx

```diff
@@ -1,23 +1,36 @@
 import React from 'react';
 import moment from 'moment-timezone';
-import { shape, bool, string } from 'prop-types';
+import { shape, bool, string, oneOfType } from 'prop-types';
 import MediaIndicatorComp from '@bbc/psammead-media-indicator';
 import pathOr from 'ramda/src/pathOr';
-import pick from 'ramda/src/pick';
-import { storyItem } from '../../../models/propTypes/storyItem';
+import { storyItem, linkPromo } from '../../../models/propTypes/storyItem';
 import formatDuration from '../../../lib/utilities/formatDuration';
 
-const MediaIndicator = ({ item, topStory, service, indexAlsos }) => {
+const getAssetContentTypes = item => {
+  const mediaContentTypes = ['audio'];
+  const type = pathOr(null, ['contentType'], item).toLowerCase();
```

Comment:
The null protection using `pathOr` is being neutered here by `toLowerCase()`.

If `item` or `contentType` is null, this code will become `null.toLowerCase()` and will throw an exception anyway.